mmv1/third_party/terraform/tests/resource_dialogflowcx_agent_test.go.erb

```diff
@@ -0,0 +1,104 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDialogflowCXAgent_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          getTestOrgFromEnv(t),
+		"billing_account": getTestBillingAccountFromEnv(t),
+		"random_suffix":   randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDialogflowCXAgent_basic(context),
+			},
+			{
+				ResourceName:            "google_dialogflow_cx_agent.foobar",
+				ImportState:             true,
+				ImportStateVerify:       true,
+			},
+			{
+				Config: testAccDialogflowCXAgent_full(context),
+			},
+			{
+				ResourceName:            "google_dialogflow_cx_agent.foobar",
+				ImportState:             true,
+				ImportStateVerify:       true,
+			},
+		},
+	})
+}
+
+func testAccDialogflowCXAgent_basic(context map[string]interface{}) string {
+	return Nprintf(`
+	resource "google_project_service" "agent_project" {
```

Comment:
Oh, I'd missed this- can you remove the `google_project_service` resource? We run tests in parallel within a single project, and they end up causing trouble because they'll turn off the API for other test cases. You can assume the API is already enabled.